structuredevent-service-cdp/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/audit/auditeventname/flow/CDPFlowResourceAuditEventConverter.java

```diff
@@ -0,0 +1,11 @@
+package com.sequenceiq.cloudbreak.structuredevent.service.audit.auditeventname.flow;
+
+import com.sequenceiq.cloudbreak.audit.model.AuditEventName;
+import com.sequenceiq.cloudbreak.structuredevent.event.cdp.CDPStructuredFlowEvent;
+
+public interface CDPFlowResourceAuditEventConverter {
```

Comment:
Is this implemented somewhere? or it is for future purposes?

Reply:
Yes it is implemented here: `public class EnvironmentFlowAuditEventNameConverter implements CDPFlowResourceAuditEventConverter {`